static/js/place/overview.tsx

```diff
@@ -32,18 +32,26 @@ interface OverviewPropType {
 
 class Overview extends React.Component<OverviewPropType> {
   render(): JSX.Element {
+    // only show ranking for places in the US, but not the US itself.
+    const shouldShowRanking = this.props.dcid.startsWith("geoId/");
```

Comment:
how about passing this in from the tile (add a prop to this component, and let the test for geoId/ stay in the tile).

Reply:
Made the change to pass this in from the tile.